cgi/product_image_upload.pl

```diff
@@ -157,16 +166,19 @@
 
 if ($imagefield) {
 
+	$response_ref->{imagefield} = $imagefield;
+
 	my $path = product_path_from_id($product_id);
 
 	$log->debug("path determined", { imagefield => $imagefield, path => $path, delete => $delete });
 
 	if ($path eq 'invalid') {
 		# non numeric code was given
 		$log->warn("no code", { code => $code });
-		my %response = ( status => 'status not ok');
-		$response{error} = "error - invalid product code: $code";
-		my $data =  encode_json(\%response);
+		$response_ref->{status} = 'status not ok';
```

Comment:
Translations? (See above)

Reply:
not sure we want to translate error messages